client/scss/utilities/_root-scope-classes.scss

```diff
@@ -293,6 +290,24 @@
   position: absolute;
 }
 
+.half-in-image {
+  margin-top: -50px;
+}
+
+.show-to-l {
```

Comment:
we have `is-hidden-s` and `is-hidden-m` ?

Reply:
Mobile first again.
And I would have to apply a few styles as it uses the `respond-between`.

Reply:
Do you think it would be best to follow convention and have a little bit extra code?

Reply:
yeah, you'd have to apply the two classes. But they're pre-existing classes

Reply:
And if we get bigger screens...